index.d.ts

```diff
@@ -0,0 +1,280 @@
+declare module '@dronedeploy/functions-wrapper' {
+
+  import * as e from "express";
+
+  /**
+   * The datastore provides an interface for accessing functionality for the DroneDeploy datastore:
+   * 
+   * https://developer.dronedeploy.com/docs/sdk-beta/datastore.html
+   */
+  interface Datastore {
+
+    /**
+     * Finds the table ID by its name
+     *
+     * @param {string} name The name of the table
+     * @param {string} slug The application slug
+     * @returns {Promise<string>} The table ID
+     */
+    findTableByName(name: string, slug: string): Promise<string>;
+
+    /**
+     * Gets a reference to the table object
+     *
+     * @param {string} tableId The table ID
+     * @returns {Promise<module:dronedeploy/functions-wrapper.Table>} The table
+     */
+    table(tableId: string): Promise<Table>;
+
+  }
+
+  /**
+   * Response returned from a datastore mutation
+   */
+  interface MutationResponse {
```

Comment:
A thought on this type --- you could also add `data?: object;` and change the name to `DatastoreResponse`.  Then it would apply to all the add/edit/upsert and getRow operations.